script/doctor-workbox.bash

```diff
@@ -3,82 +3,15 @@
 # sed expression needs $s
```

Comment:
why is this script needed at all? i thought new react-scripts handles it correctly now?

Reply:
I'm using it to move `service-worker.js` to the `books` folder in the `build`
I could try to find another way of doing it, maybe using `craco` for this

Reply:
I've changed this with craco